Fix automation re: master->main rename

cevich opened the pull request:
Ref: https://github.com/containers/image/pull/1272 and https://github.com/containers/skopeo/issues/1282

Fixes #1282

cevich commented:
Oof, [the Dockerfile build](https://quay.io/repository/skopeo/ci/build/7d2d1016-77fd-438e-948d-f8e11281c5d9) is currently broken as well, gunna needa fix that too...

cevich commented:
...okay, local testing shows the Dockerfile now builds successfully.  I added a notification to the auto-builder (on quay.io) for this image: It should now post future build-failure notices to our podman-monitor list.

mtrmac commented:
> Oof, the Dockerfile build is currently broken as well, gunna needa fix that too...

To be explicit, those are the `GO111MODULE` changes, which are otherwise unrelated?

---

The `cross` task is failing for what seems to be a Brew server failure, I have triggered a re-run.

mtrmac commented on Dockerfile:
```diff
@@ -34,6 +35,7 @@ RUN set -x \
 	&& rm -rf "$GOPATH"
 
 RUN set -x \
+    && export GO111MODULE=off \
```
A nit: Could you indent the new lines the same way as the rest, please?

cevich replied:
Oh woops, my mistake, yep I got it...

cevich commented:
@mtrmac PTAL, I fixed the whitespace and bumped up the timeout a bit to see if that helps the cross task.

mtrmac reviewed (approved):
Thanks! LGTM.

(I’m not sure the timeout is desirable, this just seems to be a Brew infrastructure issue where we instead of pre-built binaries burn CPU time re-building locally. But even if it gets CI unblocked for a week and then we can revert, that’s valuable for that week.)

mtrmac commented:
@cevich Looking at https://github.com/Homebrew/brew/issues/11119 or https://github.com/Homebrew/brew/pull/11366 , the `brew` failure seems not to be a temporary situation (although https://status.bintray.com does currently report a scheduled maintenance).

Is the `brew` version in the macOS environment new enough, or could it be updated? If not, a `brew update` before package installation might well pay off in time.

cevich commented:
> But even if it gets CI unblocked for a week and then we can revert

I'm good either way.  The same key-name can be added to individual tasks for more specific/specialized timeout values, I only stuck in a global one out of sheer laziness.

TomSweeneyRedHat commented:
LGTM

mtrmac commented:
I have filed a completely uninformed #1329 with changes from this PR + `brew update`, just to see what happens.

cevich commented:
> To be explicit, those are the GO111MODULE changes, which are otherwise unrelated?

Yes, mostly unrelated.  Though auto-builder on quay was specifically targeting the master branch.  When I fixed that, I noticed the image build has been failing for months, so, yeah, :disappointed: Anyway, I also added some monitoring, so we'll now get e-mails to the podman-monitor list any time it breaks in the future.

rhatdan commented:
LGTM
